docs/reference/rockcraft.yaml.rst

```diff
@@ -115,7 +115,7 @@ the value of ``base``.
 
 **Type**: string, in `SPDX format <https://spdx.org/licenses/>`_
```

Comment:
can you update this to say that the string `"proprietary"` is also accepted?

Reply:
sure, done